utils/guidelines_xslt/odd2html/guidelines.xsl

```diff
@@ -265,7 +265,7 @@
     <xsl:template match="tei:ref" mode="guidelines">
         <xsl:choose>
             <xsl:when test="starts-with(@target,'#')">
-                <xsl:variable name="chapter.id" select="replace(@target,'#','')" as="xs:string"/>
+                <xsl:variable name="chapter.id" select="substring-after(@target,'#')" as="xs:string"/>
```

Comment:
In general, the `replace` is more expensive, but also more robust (it still works if there is no hash). However, as the line above ensures that there is a hash to start with, it could be made even faster by just doing a `substring(@target,2)`, making it completely unnecessary to query the content…

Reply:
You're right, thanks for this!